public/templates/patWaitingRoom.html

```diff
@@ -86,41 +90,45 @@ <h3 class="command" id="StepTwo">2. Get matched with a doctor</h3>
     <p class="outlier">Now press the "Match Me" button to enter a video room. Once in the room, a doctor will meet with you shortly.
        These meetings are not recorded and are completely confidential, so feel free to discuss openly your issue. Be safe and show respect
        while using this platform.</p>
-    
+    <!-- Makes match me button  -->
     <button   class="mdc-button mdc-button--raised " disabled id="createBtn">
         <i      class="material-icons mdc-button__icon" aria-hidden="true">group_add</i>
         <span   class="mdc-button__label">Match Me</span>
     </button>
-    
+    <!-- Old match me button -->
     <button hidden class="mdc-button mdc-button--raised" disabled id="matchMe">
           <i class="material-icons mdc-button__icon" aria-hidden="true">group_add</i>
           <span class="mdc-button__label">Original Match Me</span>
     </button>
     
+    <!-- Join room button -->
     <button hidden class="mdc-button mdc-button--raised" disabled id="joinBtn">
         <i class="material-icons mdc-button__icon" aria-hidden="true">group</i>
           <span class="mdc-button__label">Join Room</span>
     </button>
     
+    <!-- Hang up button -->
     <button hidden class="mdc-button mdc-button--raised" disabled id="hangupBtn">
         <i hidden class="material-icons mdc-button__icon" aria-hidden="true">close</i>
         <span hidden class="mdc-button__label">Hangup</span>
     </button>
   </div>
+  <!-- end of div -->
   
 <!--   <div>
       <span id="currentRoom"></span>
   </div> -->
   <div id="videos"><!--Second child-->
-      <video id="localVideo" muted autoplay playsinline></video>
+      <!-- Not sure what this does -->
```

Comment:
Hmmm looks like two videos where one is hidden (from the "hidden" attribute). Knowing this is the patient waiting room, could this div contain the video chat components?